scripts/list_prompts.py

```
@@ -0,0 +1,142 @@
+import sys
+import yaml
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)